app/frontend/src/utils/timeAgo.ts

```diff
@@ -1,9 +1,42 @@
-import moment from "moment";
+const minuteMillis = 60000;
+const quaterHourMillis = 900000;
+const hourMillis = 3600000;
+const twoHourMillis = 7200000;
+const dayMillis = 86400000;
+const twoDayMillis = 172800000;
+const weekMillis = 604800000;
+const twoWeekMillis = 1209600000;
+const monthMillis = weekMillis * 4;
+const twoMonthMillis = weekMillis * 8;
+const yearMillis = 31557600000;
+const twoYearMillis = 31557600000 * 2;
+
+export function timeAgo(input: Date | string) {
+  if (input === undefined) return "";
+  const date = new Date(input);
+  const diffMillis = Date.now() - date.getTime();
 
-export function timeAgo(date: Date) {
   //the backend fuzzes times to 15 minute intervals
```

Comment:
Would be good to have some unit tests for these since we are writing our own logic/ditching `moment`

Reply:
+1

Reply:
I've added a unit test to the timeago function